annotation-tool/src/main.py

```diff
@@ -107,7 +107,13 @@ def inference(api: sly.Api, task_id, context, state, app_logger):
     ann_pred_json = api.task.send_request(state["sessionId"],
                                           "inference_image_id",
                                           data=data)
-    ann_pred = sly.Annotation.from_json(ann_pred_json, model_meta)
+    try:
+        ann_pred = sly.Annotation.from_json(ann_pred_json, model_meta)
+    except Exception as e:
+        sly.logger.warn(f"Couldn't process prediction for the image {image_id} id. Empty Annotation generated. Error: {e}")
+        sly.logger.debug(f"Prediction ann data: {ann_pred_json}")
```

Comment:
:
sly.logger.warn("Can not process predictions from serving", extra={"image_id": image_id, "details": repr(e)})
sly.logger.debug("XXX", extra={"serving_responce": "ann_pred_json"})